### PR TITLE
Ashi laura motor inertia

### PR DESCRIPTION
**Pull request by ashi24**

### CODE REVIEW

File: database/parameters.m [hunk 1]
```diff
@@ -170,7 +170,7 @@
 if (seek == "HREV1")  
     % engine locations
     retParams.X_GM = 2.244;                                                             % engine location from nose tip                     [m]
-    retParams.X_CG_full = 0.2745;                                                       % location of cg of engine from X_GM                [m]
+    retParams.X_CG_full = 0.2745;                                                       % location of cg of engine from X_GM ( to the tank tip) [m]                [m]
```
Comment (mginoya) [inline database/parameters.m]: So in this case X_GM is the length from tip of nosecone to tip of tank (ie. tip of tank is what we are considering to be the engine start point) - nothing to really change, but we could make the commenting a little clearer in the future

File: elements/pVessels/PVessel.m [hunk 2]
```diff
@@ -1,4 +1,4 @@
-classdef PVessel < Element
+ classdef PVessel < Element
```
Comment (mginoya) [inline elements/pVessels/PVessel.m]: This space should not be here - we need to get rid of this space

**Comment by mginoya** [comment]
This should not exist still - please remove this in HREv1:

image

**Comment by mginoya** [comment]
This has become quite messy. We should try to make this as clean as possible

image

**Comment by mginoya** [comment]
Naming convention here is unclear. You use the '_full' qualifier to describe cg of fuel wrt. tip of oxi tank, and to describe cg. of propellants combined wrt. tip of nosecone (see images below). We should change this to something that makes a little more sense - perhaps this is a good time to actually come up with standardized naming

First Use:
image

Second Use: 
image

**Comment by mginoya** [comment]
The outports shown below still do not match the interface that Ascent model expects.

The ascent model total cg calculation expects to take in two signals: One is a bus that contains the parameter value describing length from tip of nosecone to tip of engine as well as the cg of combined propellants wrt. to tip of engine (in this case tip of oxi tank). The other signal is the mass of the propellants. You are currently sending out the mass of propellants correctly I believe, but not the cg information (that needs to be collected in a bus). See image below of what the Ascent model expects. ie. We need to make the signal names and types the exact same as we output them from the HREv1 model (or any propulsion model) - this way, we simply have to connect the two models together with no additional changes.

Current cg outports in HREv1 model (incorrect):
image

What the Ascent Model expects:
image

**Review by mginoya — CHANGES_REQUESTED** [review]
Please see all my comments above about changes that need to be made to elementsLibrary (specifically HREv1 model which you are adding to) before we can commit. Currently, this implementation will not work with the way the new Ascent model is being developed

File: database/parameters.m (hunk 1 above)
Comment (ashi24) [reply database/parameters.m]: Thanks! I know you said nothing to change really, but I made a much clearer comment there.

**Pull request merged**